content/blog/2020-02-glue-strings-and-tidy-eval/index.Rmarkdown

```diff
@@ -0,0 +1,119 @@
+---
+title: Tidy eval now supports glue strings
+author: Lionel Henry
+date: '2020-02-11'
+slug: glue-strings-and-tidy-eval
+categories:
+  - package
+tags: [package, r-lib, tidyverse]
+photo:
+  url: https://unsplash.com/photos/pcoKkN3NsX0
+  author: Modestas Urbonas
+---
+
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>"
+)
+iris <- tibble::as_tibble(iris)
```

Comment:
Unsolicited review: I would reveal a step like this, because it affects how the code below runs.